### PR TITLE
Add missing assets command for migration

### PR DESCRIPTION
**Pull request by solivier**

### CODE REVIEW

File: developer_guide/migration/index.rst [hunk 1]
```diff
@@ -34,8 +34,9 @@ Then clean the cache, re-install assets and warmup the cache:
 
 .. code-block:: bash
 
-    rm -rf app/cache/*
-    app/console --env=prod pim:installer:assets
+    rm -rf app/cache/* ./web/bundles/* ./web/css/* ./web/js/*
+    app/console --env=prod pim:installer:assets -e=prod
```
Comment (fitn) [inline developer_guide/migration/index.rst]: You can remove `-e=prod`. Already done with `--env=prod` which is more explicit

**Comment by phaseinducer** [comment]
GTM after fix

File: developer_guide/migration/index.rst [hunk 2]
```diff
@@ -34,8 +34,9 @@ Then clean the cache, re-install assets and warmup the cache:
...
+    app/console assets:install --symlink web
```
Comment (fitn) [inline developer_guide/migration/index.rst]: symlink option should not be used for production?

**Comment by fitn** [comment]
GTM

**Pull request merged**